roles/satellite-clone/tasks/main.yml

```diff
@@ -184,8 +184,19 @@
     state: latest
   when: puppet_version|int == 4
 
+  # Because of https://projects.theforeman.org/issues/30506
+- name: Install Satellite SELinux packages for 6.8 and above
+  yum:
+    name:
+      - crane-selinux
+      - foreman-selinux
+      - katello-selinux
+    state: present
+  when: satellite_version in ["6.8"]
+
 - name: untar config files (for cloning only)
   command: tar --selinux --overwrite -xf {{ backup_dir }}/config_files.tar.gz -C / --exclude=var/lib/foreman/public
+  when: satellite_version in ["6.8"]
```

Comment:
why is this restricted to `6.8` only?

Reply:
Because on 6.7 and below the package is a dep of satellite and would
already be installed.

Reply:
I understand about the new task `name: Install Satellite SELinux packages for 6.8 and above`

But this step below, if I understand correctly is general for all versions. If you restrict it to 6.8 only, older versions won't untar isn't it?
```
- name: untar config files (for cloning only)
  command: tar --selinux --overwrite -xf {{ backup_dir }}/config_files.tar.gz -C / --exclude=var/lib/foreman/public`
```

Reply:
Oh my bad, I must have failed during copy/paste. me-- I will fix that and push up my changes and do another dolly run.

Reply:
Nice catch

Reply:
@sthirugn Updated. Starting dolly run

Reply:
thanks